### PR TITLE
Update hotbar controller for the new RACE changes in Heroes.

### PR DESCRIPTION
**Pull request by Kainzo**
Please accept this fork when the new version of Heroes is released (tenatively 06/01/18)

### CODE REVIEW

File: src/main/java/com/elmakers/mine/bukkit/heroes/HotbarController.java [hunk 1]
```diff
@@ -331,22 +331,30 @@ private void addSkills(Hero hero, HeroClass heroClass, Collection<String> skillS
         if (hero == null) return emptySkillList;
 
         HeroClass heroClass = hero.getHeroClass();
-        HeroClass secondClass = hero.getSecondaryClass();
+        HeroClass secondClass = hero.getSecondaryClass();();
```
Comment (NathanWolf) [inline src/main/java/com/elmakers/mine/bukkit/heroes/HotbarController.java]: This looks weird, does it compile? Is (); a valid statement in Java?

File: src/main/java/com/elmakers/mine/bukkit/heroes/HotbarController.java [hunk 2]
```diff
@@ -331,22 +331,30 @@ private void addSkills(Hero hero, HeroClass heroClass, Collection<String> skillS
...
+        HeroClass raceClass = hero.getSecondaryClass();
         Set<String> primarySkills = new HashSet<>();
         Set<String> secondarySkills = new HashSet<>();
+        Set<String> raceSkills = new HashSet<>();
         addSkills(hero, heroClass, primarySkills, showUnuseable, showPassive);
         addSkills(hero, secondClass, secondarySkills, showUnuseable, showPassive);
+        addSkills(hero, raceClass, raceySkills, showUnuseable, showPassive);
```
Comment (NathanWolf) [inline src/main/java/com/elmakers/mine/bukkit/heroes/HotbarController.java]: Should raceySkills be raceSkills here?

File: src/main/java/com/elmakers/mine/bukkit/heroes/HotbarController.java [hunk 3]
```diff
@@ -331,22 +331,30 @@ private void addSkills(Hero hero, HeroClass heroClass, Collection<String> skillS
         if (hero == null) return emptySkillList;
 
         HeroClass heroClass = hero.getHeroClass();
-        HeroClass secondClass = hero.getSecondaryClass();
+        HeroClass secondClass = hero.getSecondaryClass();();
+        HeroClass raceClass = hero.getSecondaryClass();
```
Comment (NathanWolf) [inline src/main/java/com/elmakers/mine/bukkit/heroes/HotbarController.java]: I'm guessing this is supposed to be some new method call. Is there like a getRaceClass()? Or is it getTertiaryClass()? Something else?

File: src/main/java/com/elmakers/mine/bukkit/heroes/HotbarController.java (hunk 1 above)
Comment (Kainzo) [reply src/main/java/com/elmakers/mine/bukkit/heroes/HotbarController.java]: Oh geez... I dont know what happened with this, I guess I copy/pasted wrong. Fixing.

File: src/main/java/com/elmakers/mine/bukkit/heroes/HotbarController.java (hunk 2 above)
Comment (Kainzo) [reply src/main/java/com/elmakers/mine/bukkit/heroes/HotbarController.java]: A fat fingered typo, thanks for the review.

File: src/main/java/com/elmakers/mine/bukkit/heroes/HotbarController.java (hunk 3 above)
Comment (Kainzo) [reply src/main/java/com/elmakers/mine/bukkit/heroes/HotbarController.java]: Checking the method now.  It's actually hero.getRaceClass

**Comment by Kainzo** [comment]
Alright... I think I fixed everything.  Someone changed this for me, I assumed they did a code review before they submitted it - but its completed now.

**Pull request merged**